Foundation/NSData.swift

```diff
@@ -1138,3 +1138,7 @@ extension NSData : _StructTypeBridgeable {
         return Data._unconditionallyBridgeFromObjectiveC(self)
     }
 }
+
+internal func _CFSwiftDataCreateCopy(_ data: AnyObject) -> Unmanaged<AnyObject> {
+    return Unmanaged<AnyObject>.passRetained((data as! NSDictionary).copy() as! NSObject)
```

Comment:
I don’t think `as! NSDictionary` is correct. Shouldn’t it be `as! NSData`?

Reply:
A copy-and-paste error. I postponed adding tests due to the size, but this may merit one. Thanks!